### PR TITLE
Fixes for XLA to ATEN generator script

### PR DESCRIPTION
**Pull request by asuhan**
Include missing header, use c10::Backend instead of Backend.

### CODE REVIEW

File: scripts/gen.py [hunk 1]
```diff
@@ -68,6 +68,7 @@
 _CPP_HEADER = """// Autogenerated file by {}. Do not edit directly!
 
 #include "aten_xla_bridge.h"
+#include <ATen/ExtensionBackendRegistration.h>
```
Comment (dlibenzi) [inline scripts/gen.py]: Let's add this when the PR goes in, as otherwise this won't build.

Comment (asuhan) [reply scripts/gen.py]: `Backend::XLA` doesn't exist yet either. How about not generating this when `generate_class` is set?

Comment (asuhan) [reply scripts/gen.py]: Actually, this header wouldn't be generated in that mode, so we're good.

Comment (dlibenzi) [reply scripts/gen.py]: SG

File: scripts/gen.py [hunk 2]
```diff
@@ -642,7 +643,7 @@ def generate_registrations(fgens):
   code = 'void RegisterAtenTypeFunctions() {\n'
   for fgen in fgens:
     code += (
-        '  at::register_extension_backend_op(\n    Backend::XLA,\n    "{}",\n'
+        '  at::register_extension_backend_op(\n    c10::Backend::XLA,\n    "{}",\n'
```
[the remark this answers is not in the document]
Comment (asuhan) [reply scripts/gen.py]: Done.

**Pull request merged**